src/msg/async/PosixStack.cc

```diff
@@ -177,7 +177,7 @@ class PosixServerSocketImpl : public ServerSocketImpl {
       handler(h), _fd(f) {}
   int accept(ConnectedSocket *sock, const SocketOptions &opts, entity_addr_t *out, Worker *w) override;
   void abort_accept() override {
-    ::close(_fd);
+    ::shutdown(_fd, SHUT_RD);
```

Comment:
please move this change into a separated commit.

Reply:
OK